pkg/k8shandler/secret_test.go

```diff
@@ -0,0 +1,265 @@
+package k8shandler
+
+import (
+	"reflect"
+	"testing"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	rbac "k8s.io/api/rbac/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestClusterLoggingRequest_CreateOrUpdateSecretNoop(t *testing.T) {
+	clusterRoleBinding := &rbac.ClusterRoleBinding{}
```

Comment:
Shouldn't this be the object you want to return when you do a `get`? ref `https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/fake/client.go#L58`  at a minimum it should likely be a `secret` since that is what we are testing.  This applies throughout the PR.  I'm surprised the test actually pass

Reply:
@jcantrill, I borrowed this code to generate clusterLoggingRequest... :p  Could you please tell me what &rbac.ClusterRoleBinding{} should be replaced with?  Could it be `secret` from https://github.com/nhosoi/cluster-logging-operator/blob/bz1745394/pkg/k8shandler/secret_test.go#L31?  Thanks!